airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftSqlOperations.java

```diff
@@ -4,41 +4,67 @@
 
 package io.airbyte.integrations.destination.redshift;
 
+import static io.airbyte.db.jdbc.JdbcUtils.getDefaultSourceOperations;
+import static java.util.stream.Collectors.toSet;
+
 import com.fasterxml.jackson.databind.JsonNode;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.db.jdbc.JdbcDatabase;
 import io.airbyte.integrations.base.JavaBaseConstants;
 import io.airbyte.integrations.destination.jdbc.JdbcSqlOperations;
 import io.airbyte.integrations.destination.jdbc.SqlOperations;
 import io.airbyte.integrations.destination.jdbc.SqlOperationsUtils;
+import io.airbyte.integrations.destination.jdbc.WriteConfig;
+import io.airbyte.integrations.destination.redshift.enums.RedshiftDataTmpTableMode;
 import io.airbyte.protocol.models.AirbyteRecordMessage;
 import java.nio.charset.StandardCharsets;
 import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Set;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class RedshiftSqlOperations extends JdbcSqlOperations implements SqlOperations {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RedshiftSqlOperations.class);
   protected static final int REDSHIFT_VARCHAR_MAX_BYTE_SIZE = 65535;
+  private static final String SELECT_ALL_TABLES_WITH_NOT_SUPER_TYPE_SQL_STATEMENT = """
+      select tablename, schemaname
+      from pg_table_def
+      where
+      schemaname = '%s'
+      and "column" = '%s'
+      and type <> 'super'
+      and tablename like '%%raw%%'""";
```

Comment:
Is grepping just "raw" sufficient? it is really generic word to look for... Not just airbyte is using that in the naming of table.

maybe looking for `%%airbyte_raw%%` would be more specific? and add some conditions on the column that we typically use? (`ab_id`, `airbyte_data`, etc)

I am guessing it would be pretty bad if you change datatypes from a user schema that is not related to airbyte...

Reply:
Done. Thanks.